src/Style/Sass/base/_base.scss

```diff
@@ -15,15 +15,7 @@ nav {
   display: block !important;
 }
 
-.hide {
```

Comment:
@MCFrank16 are you sure these styles aren't being used by someone else?

Reply:
we changed these with @truestbyheart so this is the proper one

Reply:
and I received those changes while I was rebasing